setup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+from setuptools import setup, find_packages
+
+setup(
+    name='superboucle',
+    version='1.2.0',
+    packages=find_packages(),
+    author='Vampouille',
+    author_email='superboucle@nura.eu',
+    description='Loop application synced with jack transport',
+    long_description=open('readme.md').read(),
+    install_requires=["SoundFile>=0.10", "PyQt5>=5.11", "numpy>=1.16"],
+    include_package_data=True,
+    url='http://superboucle.nura.eu',
+    classifiers=[
+        "Programming Language :: Python",
+        "Development Status :: 4 - Beta",
+        "Intended Audience :: End Users/Desktop",
+        "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+        "Natural Language :: English",
+        "Operating System :: OS Independent",
+        "Programming Language :: Python :: 3.7",
+        "Topic :: Multimedia :: Sound/Audio :: Capture/Recording",
+        "Topic :: Multimedia :: Sound/Audio :: MIDI",
+    ],
+        # C'est un système de plugin, mais on s'en sert presque exclusivement
```

Comment:
Would be great to add these comments in english

Reply:
I will remove it because it came from an example